ext/nnstreamer/tensor_filter/meson.build

```diff
@@ -30,6 +32,38 @@ if get_option('enable-nnfw-runtime')
     dependencies: nnstreamer_filter_nnfw_deps)
 endif
 
+if get_option('enable-armnn')
+  filter_sub_armnn_sources = [
+    'tensor_filter_armnn.cc'
+  ]
+
+  nnstreamer_filter_armnn_sources = []
+  foreach s : filter_sub_armnn_sources
+    nnstreamer_filter_armnn_sources += join_paths(meson.current_source_dir(), s)
+  endforeach
+
+  armnn_dep = dependency('armnn', required: true)
+  nnstreamer_filter_armnn_deps = [glib_dep, gst_dep, nnstreamer_dep, armnn_dep]
+
+  armnn_plugin_lib = shared_library('nnstreamer_filter_armnn',
+    nnstreamer_filter_armnn_sources,
+    cpp_args: ['-Wno-sign-compare'],
+    dependencies: nnstreamer_filter_armnn_deps,
+    install: true,
+    install_dir: filter_subplugin_install_dir
+  )
+  static_library('nnstreamer_filter_armnn',
+    nnstreamer_filter_armnn_sources,
+    cpp_args: ['-Wno-sign-compare'],
+    dependencies: nnstreamer_filter_armnn_deps,
+    install: false,
+    install_dir: filter_subplugin_install_dir
```

Comment:
let's check install directory for static libraries (filter_subplugin_install_dir : nnstreamer_libdir)

Reply:
Updated.